### PR TITLE
feat: enable comet shuffle manager for comet shell

### PR DESCRIPTION
**Pull request by zuston**
## Which issue does this PR close?

Closes #.

## Rationale for this change

Enable the comet shuffle manager to activate some native operators for comet spark shell, like native `collectLimitExec`

### CODE REVIEW

File: bin/comet-spark-shell [hunk 1]
```diff
@@ -81,4 +81,6 @@ RUST_BACKTRACE=1 $SPARK_HOME/bin/spark-shell \
   --conf spark.comet.enabled=true \
   --conf spark.comet.exec.enabled=true \
   --conf spark.comet.exec.all.enabled=true \
+  --conf spark.comet.exec.shuffle.enabled=true \
+  --conf spark.shuffle.manager=org.apache.spark.sql.comet.execution.shuffle.CometShuffleManager\
```
Comment (sunchao) [inline bin/comet-spark-shell:85]: nit: add a space before `\`

**Review by sunchao — APPROVED** [review]
Makes sense.

**Comment by sunchao** [comment]
Merged, thanks!

**Review by viirya — COMMENTED** [review]
Looks good.

**Pull request merged**